normalizer.py

```diff
@@ -0,0 +1,137 @@
+import string
+
+import inflect
+
+
+class Normalizer:
+    AMERICAN_SPELLINGS = {
```

Comment:
I'm a proponent of keeping large hard-coded data structures in separate files. It creates a separation between data and code, which is useful in a host of different ways.

Reply:
In general I agree. I would rather not change it here, though, since this repo is quite small and another file might make it look unnecessarily messy (without giving this repo a proper folder structure).

Reply:
The cost of adding a file to the repo is outweighed by the maintainability in my opinion. If you're confident that we'll be modifying this list very infrequently in the future, I could live with it.